app/lib/gpw/client.rb

```diff
@@ -102,5 +107,36 @@ def parse_money_response_nc(response)
         }
       end
     end
+
+    def fetch_shareholders(isin)
+      @client.post('https://www.gpw.pl/ajaxindex.php',
+        {
+          start: 'shareholdersTab',
+          format: 'html',
+          action: 'GPWListaSp',
+          gls_isin: isin,
+        },
+        {
+          'Accept' => '*/*',
+          'DNT' => '1',
+          'User-Agent' => 'Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:99.0) Gecko/20100101 Firefox/99.0',
```

Comment:
do you need it?

Reply:
The `'User-Agent'` is needed, others will delete